homeassistant/components/iotty/const.py

```diff
@@ -0,0 +1,16 @@
+"""Constants for the iotty integration."""
+
+from datetime import timedelta
+import logging
+
+DOMAIN = "iotty"
+
+OAUTH2_AUTHORIZE = "https://auth.iotty.com/.auth/oauth2/login"
+OAUTH2_TOKEN = "https://auth.iotty.com/.auth/oauth2/token"
+OAUTH2_CLIENT_ID = "hass-iotty"
+
+IOTTYAPI_BASE = "https://homeassistant.iotty.com/"
+
+
+LOGGER = logging.getLogger(__package__)
+UPDATE_INTERVAL = timedelta(seconds=10)
```

Comment:
I checked a bit what the other integrations are doing, and I found it as a reasonable value. It can certainly be bigger. Do you have any recommendations on this?

Reply:
Hi, I have changed this to be 30s, I hope this is fine